models/uniswap/ethereum/uniswap_v3_ethereum_trades.sql

```diff
@@ -65,6 +66,7 @@ SELECT
     ,coalesce(dexs.taker, tx.from) AS taker -- subqueries rely on this COALESCE to avoid redundant joins with the transactions table
     ,dexs.maker
     ,dexs.project_contract_address
+    ,dexs.fee
```

Comment:
nice & simple. we'll think through this better when moving on with new `dex.trades` redesign (more to come soon!), but for short term, maybe add a sql comment on this field to indicate it's unique to this model and doesn't feed downstream spells?

Reply:
good idea, added! look forward to the redesign!